package-builder/README.md

```diff
@@ -43,19 +43,20 @@ We currently support the latest patch version of 5.6, 7.0, 7.1, and 7.2. See
 * SeasLog
 * stackdriver_debugger
 * stomp
-* suhosin
+* suhosin (5.6)
```

Comment:
Since `5.6` isn't supported anymore wouldn't this be removed?

Reply:
I have no idea what the policy is regarding removing support for previous versions.
If it was not removed when support was added for 7.+ then why should I be the one to remove it?